### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by psbaruah**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please remove `<` and `>` they are not needed.

**Review by wdhdev — APPROVED** [review]

File: domains/shl.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
 }
 {
   "record": {
-    "CNAME": "<go.tldr.ml>"
+    "CNAME": "go.tldr.ml"
```
[the remark this answers is not in the document]
Comment (AkumaX1) [reply domains/shl.json:9]: go.tldr.ml is THE website .

[the remark this answers is not in the document]
Comment (AkumaX1) [reply domains/shl.json:9]: it just has passoword .

[the remark this answers is not in the document]
Comment (WilliamDavidHarrison) [reply domains/shl.json:9]: @AkumaX1 Please provide the password, so we can check the content of your site.

**Pull request closed without merging**